Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -298,6 +299,7 @@ local questsToBlacklistBySoDPhase = {
         [90147] = true, -- Hiding Paladin Hand of Reckoning Westfall for now as there are too many icons
         [90148] = true, -- Hiding Paladin Exorcist Duskwood for now as there are too many icons
         [90154] = true, -- Hiding Druid Lacerate Westfall for now as there are too many icons
+        [90159] = true, -- Hiding Priest Void Plague Tirisfal Glades for now as there are too many icons
```

Comment:
Instead of hiding this quest, fix the spawns of Gillgar

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5486/commits/6efc55974835f048917426fd280eb7f1d4d22c52

And added the 'Elite' flag.